Assets/Mirror/Runtime/NetworkIdentity.cs

```diff
@@ -671,24 +628,9 @@ internal bool OnCheckObserver(NetworkConnection conn)
             return true;
         }
 
-        internal void OnNetworkDestroy()
+        internal void NetworkDestroy()
```

Comment:
Is this called anywhere?  It is not doing anything

Reply:
added as listener in awake and invoked from either NetworkClient or NetworkServer